src/api/cpp/cvc5.cpp

```diff
@@ -1877,9 +1886,193 @@ size_t Op::getNumIndices() const
       break;
     default: CVC5_API_CHECK(false) << "Unhandled kind " << kindToString(k);
   }
+  return size;
+}
+
+Term Op::operator[](size_t index) const
```

Comment:
Shouldn't we just use getIndicesHelper() for this method?  Then the implementation here would only be 2 lines?

Reply:
Either that or the opposite (getIndices uses getIndexHelper or something).

Reply:
Yes, there is some redundancy here. Personally I prefer to have getIndicesHelper only. But  @4tXJ7f  commented the following for performance. 
```
I don't think that we should construct a vector and then index into that, though. Ideally, the helper just takes the index of the index and returns it depending on the kind.
```
For the same performance reason, `getIndexHelper` would be called `n` times for `getIndices` if there are `n` indices which is inefficient. 
So I implemented them both separately to optimize both of them. However, I agree maintaining them would be annoying with new indices.

Reply:
After offline discussion, we decided to implement `getIndexHelper` which is called by both operator[] and getIndices90